src/hdmf/utils.py

```diff
@@ -668,8 +668,6 @@ def func_call(*args, **kwargs):
                 return func(**pargs)
 
         _rtype = rtype
-        if isinstance(rtype, type):
-            _rtype = rtype.__name__
```

Comment:
I think now that I updated the `hdmf.utils._builddoc` to convert types to strings, it should be ok to remove this - it caused some reference warnings. But I'm not sure if there were any other implications/reasoning to these lines that I was missing

Reply:
This looks good.